turbinia/config/turbinia_config_tmpl.py

```diff
@@ -230,3 +230,13 @@
 # Email address and password
 EMAIL_ADDRESS = 'example@address.com'
 EMAIL_PASSWORD = 'Hunter2'
+
+###############################################################################
+#                           Prometheus Config
+#
+# These options are required for customizing the prometheus configuration
+###############################################################################
+
+# Prometheus listen address and port
+PROMETHEUS_ADDR = '0.0.0.0'
```

Comment:
I wonder if there is somewhere in the documentation that we can mention this.  I can't think of a great place off the top of my head, but it seems like a good thing to document somewhere.

Reply:
Good point, added some text to the operational documentation which also contains the stackdriver configuration info.